src/components/inventory/index.js

```diff
@@ -43,8 +49,19 @@ class InventoryScene extends React.PureComponent {
   }
 }
 
-const mapStateToProps = state => ({
-  items: selectors.getItemsState(state)
-});
+const mapDispatchToProps = dispatch => {
+  return {
+    sellItem: (id, item) => dispatch(sellItem(id, item))
```

Comment:
:bug: `sellItem: (id, item) => dispatch(sellItem.sellItem(id, item))`

Best to rename action so that you can call it as `itemAction.sell(...)`